Login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Trainwise - Login page</title>
+
+</head>
+
+<body>
+
+    <h1>Trainwise</h1>
+
+    <h2>Login to your account</h2>
+    <form>
```

Comment:
form needs an `action` attribute - the URL on the backend that will process the request. Also a `method` attribute (`GET` or `POST`) which should be `POST` in this case since we don't want the user information appear in the URL (query string `?username=&password=`)

Reply:
Thanks,

Will read on this and if I'll have questions I'll ask :+1: